public/network_vis_controller.js

```diff
@@ -488,7 +463,7 @@ module.controller('KbnNetworkVisController', function ($scope, $sce, $timeout, P
                     });
 
                   // NODE-RELATION Type
-                } else if (secondBucketId) {
+                } else if (secondBucketId && !firstSecondBucketId) {
```

Comment:
This is the block we were entering when node-node-relation was selected. Added a check to ensure there's no secondary node.